certified-connectors/Tribal - SITS/README.md

```diff
@@ -0,0 +1,36 @@
+# Tribal - SITS
```

Comment:
Please check the below documentation and update all the required sections. 

https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md

Reply:
done